<launch>
  <node pkg="ccny_rgbd" type="dense_tracker_node" name="dense_tracker" 
    output="screen">

    <param name="fixed_frame"        value="odom"/>
    <param name="odom_frame"         value="base_link"/>

    <param name="use_additive_model"      value="true"/>

    <param name="use_vgf_data_filter"         value="true"/>
    <param name="vgf_data_res"                value="0.10"/>
    <param name="vgf_data_range"              value="3.5"/>

    <param name="octree_resolution"      value="0.03"/>

    <param name="use_color"                value="false"/>
    <param name="max_color_diff"           value="40.0"/>
    <param name="max_distance"           value="0.5"/>
    <param name="max_iterations"         value="10"/>
    <param name="max_iterations_inner"   value="20"/>
    <param name="nn_count"               value="10"/>
    <param name="epsilon"                value="5e-3"/>
    <param name="epsilon_rot"            value="2e-2"/>


  </node>

</launch>
